### PR TITLE
data verification tool

### PR DESCRIPTION
**Pull request by ooooooops**
Verify whether the collected data meets the requirement of localization map making. @anyong298 @deidaraho

### CODE REVIEW

File: modules/map/tools/map_datachecker/alignment.hpp [hunk 1]
```diff
@@ -0,0 +1,151 @@
+/******************************************************************************
+ * Copyright 2017 The Apollo Authors. All Rights Reserved.
```
Comment (deidaraho) [inline modules/map/tools/map_datachecker/alignment.hpp]: Copy right should be 2019

File: docker/build/installers/py27_requirements.txt [hunk 2]
```diff
@@ -4,8 +4,12 @@
 
 # Google infras.
 glog
-protobuf == 3.1
+protobuf == 3.3
```
Comment (deidaraho) [inline docker/build/installers/py27_requirements.txt]: is protobuf 3.3 a must dependence

File: modules/map/tools/map_datachecker/alignment_agent.hpp [hunk 3]
```diff
@@ -0,0 +1,202 @@
+/******************************************************************************
+ * Copyright 2017 The Apollo Authors. All Rights Reserved.
```
Comment (deidaraho) [inline modules/map/tools/map_datachecker/alignment_agent.hpp]: 2019 copyright

File: modules/map/tools/map_datachecker/alignment_agent.hpp [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+using EIGHTROUTE_REQUEST_TYPE
+    = const apollo::hdmap::EightRouteRequest;
+using EIGHTROUTE_RESPONSE_TYPE
+    = apollo::hdmap::EightRouteResponse;
+
+template<typename ALIGNMENT_TYPE, typename REQUEST_TYPE, typename RESPONSE_TYPE>
+class AlignmentAgent {
+ public:
+    AlignmentAgent(
+        std::shared_ptr<JSonConf> sp_conf,
```
Comment (deidaraho) [inline modules/map/tools/map_datachecker/alignment_agent.hpp]: please follow the google coding style, only 2 space in front.

**Comment by jinghaomiao** [comment]
@ooooooops please sign the CLA

File: docker/build/installers/py27_requirements.txt (hunk 2 above)
Comment (ooooooops) [reply docker/build/installers/py27_requirements.txt]: 目前docker中grpc和protobuf的c++版本分别是1.14和3.3。python版的grpc使用1.14版本时，使用3.1版本的protobuf会出现不兼容的问题。为解决兼容问题以及和c++版本保持一致，希望能把protobuf升级成3.3版本。

File: modules/map/tools/map_datachecker/alignment.hpp (hunk 1 above)
Comment (ooooooops) [reply modules/map/tools/map_datachecker/alignment.hpp]: 好的，下次提交的时候修改

File: modules/map/tools/map_datachecker/alignment_agent.hpp (hunk 4 above)
Comment (ooooooops) [reply modules/map/tools/map_datachecker/alignment_agent.hpp]: 好的，下次提交的时候修改

**Comment by deidaraho** [comment]
please check why TeamCity build failed.

**Comment by jinghaomiao** [comment]
The ticket has not been updated for a while. It's being closed. Please submit a new one.

**Pull request closed without merging**